src/main/java/NullabilityRefactoring.java

```diff
@@ -0,0 +1,372 @@
+import org.eclipse.jdt.core.dom.*;
+import org.eclipse.jdt.core.dom.rewrite.ASTRewrite;
+import java.util.*;
```

Comment:
It's generally considered bad style in Java to use wildcard imports, because it pollutes the namespace.